website/templates/profile/personal_tokens_detail.mako

```diff
@@ -65,9 +65,9 @@
                         <div class="bg-danger f-w-xl token-warning">This is the only time your token will be displayed.</div>
                         <label class="f-w-xl">Token ID</label>
                         <i class="fa fa-info-circle text-muted" data-bind="tooltip: {title:'ID used to authenticate with this token. This will be shown only once.',        placement: 'bottom'}"></i>
-                        <span data-bind="text: token_id" id="token-id-text"></span>
+                        <span data-bind="text: token_id"></span>
                         <div>
-                            <button type="button" class="btn btn-primary" data-clipboard-target="#token-id-text" id="copy-button"><i class="fa fa-copy"></i> Copy to clipboard</button>
+                            <button type="button" class="btn btn-primary" data-bind="clipboard: token_id()" id="copy-button"><i class="fa fa-copy"></i> Copy to clipboard</button>
```

Comment:
Minor: We can probably tone down the "visual weight" of this button. What if we make it grey (`.btn-default`)?